src/stores/api/handlers/eth1.go

```diff
@@ -43,6 +43,7 @@ func (h *Eth1Handler) Register(r *mux.Router) {
 	r.Methods(http.MethodPost).Path("/{address}/sign-quorum-private-transaction").HandlerFunc(h.signPrivateTransaction)
 	r.Methods(http.MethodPost).Path("/{address}/sign-eea-transaction").HandlerFunc(h.signEEATransaction)
 	r.Methods(http.MethodPost).Path("/{address}/sign-typed-data").HandlerFunc(h.signTypedData)
+	r.Methods(http.MethodPost).Path("/{address}/sign-eip191-data").HandlerFunc(h.signEIP191Data)
```

Comment:
it should be called `sign-data` only and the old `sign-data` rename to `sign-hash`

Reply:
Ok